test/e2e/protocol/proposer.test.mjs

```diff
@@ -26,14 +26,19 @@ const web3Client = new Web3Client();
 describe('Basic Proposer tests', () => {
   before(async () => {
     for (const prop of testProposers) {
+      console.log('INITIALISING');
```

Comment:
just a shout out to remove these ```console.logs``` :smile:

Reply:
Oh yes, sorry, I forgot those.